app/src/main/java/skills/future/planer/ui/month/CustomMultipleDotSpan.java

```diff
@@ -0,0 +1,81 @@
+package skills.future.planer.ui.month;
+
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.text.style.LineBackgroundSpan;
+
+/**
+ * Class implements LineBackgroundSpan and draws dots under dates
+ *
+ * @author Paweł Helisz
+ * @version 0.2.2
+ * @since 0.2.1
+ */
+public class CustomMultipleDotSpan implements LineBackgroundSpan {
+
+    /**
+     * dot radius
+     */
+    private final float radius;
+    /**
+     * number of dots under date
+     */
+    private final int dots_number;
+    /**
+     * colors of dots
+     */
+    int[] color = {
+            Color.rgb(0, 0, 255),
+            Color.rgb(0, 255, 0),
+            Color.rgb(255, 0, 0),
+            Color.rgb(255, 0, 255),
+    };
```

Comment:
czemu nie wykorzystujesz kolorów które Jarek wcześniej zdefiniował?

Reply:
Bo kolory kropek mają być inne niż reszta, jest więcej opcji do rozwoju tego dalej ze względu na kategorie. To nie jest wersja finalna przypominam